core/system/src/mpu/vmpu_armv7m.c

```diff
@@ -717,3 +717,42 @@ void vmpu_arch_init_hw(void)
     );
 #endif /* !defined(UVISOR_IN_STANDALONE_SRAM) */
 }
+
+int vmpu_is_region_size_valid(uint32_t size)
+{
+    /* Get the MSB of the size */
```

Comment:
I don't think adding a period contributes at all to making this code more understandable or maintainable or more secure.
This is an internal comment, not API documentation.